Gemfile.lock

```diff
@@ -70,8 +70,5 @@ DEPENDENCIES
   sinatra!
   yajl-ruby
 
-RUBY VERSION
```

Comment:
I think this change was added because I ran `bundle install` before starting work. Not sure whether it's needed or not? @tmtmtmtm @chrismytton